app/app/my-markets/page.tsx

```
@@ -374,48 +380,75 @@ const MyMarketsPage: FC = () => {
     return (
-      <main className="mx-auto max-w-5xl px-4 py-24 text-center">
-        <div className="mx-auto max-w-md rounded-[4px] border border-[#1a1a1f] bg-[#111113] p-8">
-          <h1 className="text-xl font-bold text-white">your markets</h1>
-          <p className="mt-2 text-sm text-[#71717a]">connect your wallet to see what you&apos;ve built.</p>
-        </div>
-      </main>
+      <div className="min-h-[calc(100vh-48px)] relative">
+        <div className="absolute inset-x-0 top-0 h-48 bg-grid pointer-events-none" />
+        <main className="relative mx-auto max-w-4xl px-4 py-10">
+          <div className="mb-2 text-[10px] font-medium uppercase tracking-[0.25em] text-[var(--accent)]/60">
+            // admin
```

Biome (2.3.13)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)